dcp_prototype/backend/wrangling/migrations/common/dataset_metadata.py

```diff
@@ -50,6 +50,18 @@ def process(self):
         self.process_dissociation_protocol()
         self.process_library_preparation_protocol()
         self.process_enrichment_protocols()
+        self.process_cell_suspension()
+        self.process_organoid()
+        self.process_cell_line()
+
+        # NOTE: this may not be the proper way to get the biosample_category.
+        # This need to be reviewed and checked.
+        if self.entity_data.get("organoid"):
+            self.project.biosample_categories.append("organoid")
+        elif self.entity_data.get("cell_line"):
+            self.project.biosample_categories.append("cell line")
+        elif self.entity_data.get("specimen_from_organism"):
+            self.project.biosample_categories.append("primary tissue")
```

Comment:
I think the category might be "primary specimen" instead of "primary tissue".

Reply:
The biosample prep object document lists it as "primary tissue".